client/api/volume/v1alpha1/api.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package v1alpha1;
+
+service Volume {
+    // ListVolumesOnDisk returns the volume IDs (in \\.\Volume{GUID} format) for
+    // all volumes on a Disk device
+    rpc ListVolumesOnDisk(ListVolumesOnDiskRequest) returns (ListVolumesOnDiskResponse) {}
```

Comment:
what do you mean `all volumes on a Disk device`? volume means `partition` of a disk on windows?

Reply:
In the context here, I think volume can be considered equal to partition which is already formatted. @ddebroy - thoughts ?

Reply:
If we are formatting the partitions with -UseMaximumSize we will have only a single volume present. If we have to start supporting multiple partitions then we need to start looking at how to specify those format requirements at the beginning of the workflow.

Reply:
/cc @jingxu97

Reply:
get-disk | get-partition| get-volume is what works :-)